content/en/docs/concepts/security/multi-tenancy.md

```diff
@@ -422,8 +422,8 @@ Priority and Fairness, is also recommended.
 
 Namespace isolation is well-supported by Kubernetes, has a negligible resource cost, and provides
 mechanisms to allow tenants to interact appropriately, such as by allowing service-to-service
-communication. However, it can be difficult to configure, and doesn't apply to Kubernetes
-resources that can't be namespaced, such as Custom Resource Definitions, Storage Classes, and Webhooks.
+communication. However, it can be difficult to configure, and it doesn't apply to Kubernetes resources
+that aren't namespaced, such as APIServices, StorageClasses, or ClusterTrustBundles.
```

Comment:
Since the confusion is common to some users, we should not shy away from mentioning CustomResourceDefinitions. We may instead want to emphasize that CustomResourceDefinitions are cluster scoped, but custom resources created from CRDs do not necessarily have to be cluster scoped.
Otherwise, what's the point of this PR then?

Reply:
Thank you for the feedback, @tengqm and @sftim! I've updated the statement to maintain mention of CustomResourceDefinitions (CRDs) for clarity, specifying that CRDs are cluster-scoped, but the custom resources they define can be namespaced. Here's the updated text:

"However, it can be difficult to configure, and it doesn't apply to Kubernetes resources that aren't namespaced, such as APIServices, StorageClasses, ClusterTrustBundles, or CustomResourceDefinitions (CRDs) themselves, though the custom resources they define can be namespaced."

Let me know if this looks good, and I'll proceed with the updates.